docs/add_content_and_edit_views.md

```diff
@@ -0,0 +1,266 @@
+}
+```
+
+Remember to dump your assets before trying to run this code for the first time.
+After this is done, we should be able to edit Content Item containing our Tweet Field Type. 
+
+## Implement content view
+
+The steps needed to create content view are analogical to the ones needed to create edit view.
+
+### Template file
+
+First, we need to add template file that will be responsible for displaying content of our Field Type:
+
+```html
+//Resources/public/templates/fields/view/tweet.hbt
+
+<div class="ez-fieldview-row pure-g">
+    <div class="ez-fieldview-label pure-u">
+        <p class="ez-fieldview-name"><strong>{{ translate_property fieldDefinition.names }}</strong></p>
+    </div>
+    <div class="ez-fieldview-value pure-u"><div class="ez-fieldview-value-content">
+        {{#if isEmpty }}
+            {{translate 'fieldview.field.empty' 'fieldview'}}
+        {{else}}
+            {{ value.url }}
+        {{/if}}
+    </div></div>
+</div>
+```
+
+### Javascript view file
+
+Next, we need to implement Javascript file that will be responsible for hendling the template:
+
+```javascript
+//Resources/public/js/views/fields/ez-tweet-view.js
+
+YUI.add('ez-tweet-view', function (Y) {
+    "use strict";
+    /**
+     * Provides the Tweet field view
+     *
+     * @module ez-tweet-view
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Tweet field view
+     *
+     * @namespace eZ
+     * @class TweetView
+     * @constructor
+     * @extends eZ.FieldView
+     */
+    Y.eZ.TweetView = Y.Base.create('tweetView', Y.eZ.FieldView, [], {
+        /**
+         * Returns the value to be used in the template. If the value is not
+         * filled, it returns undefined otherwise an object with a `url` entry.
+         *
+         * @method _getFieldValue
+         * @protected
+         * @return Object
+         */
+        _getFieldValue: function () {
+            var value = this.get('field').fieldValue, res;
+
+            if ( !value || !value.url ) {
+                return res;
+            }
+            res = {url: value.url};
+
+            return res;
+        }
+    });
+
+    Y.eZ.FieldView.registerFieldView('eztweet', Y.eZ.TweetView);
+});
+```
+
+### Registering the view
+
+We also need to add to the already created configuration information about our freshly created files. To achieve this, let's add the following snippet to the `yui.yml` file:
+```yaml
+                tweetview-ez-template:
+                    type: 'template'
+                    path: bundles/ezsystemstweetfieldtype/templates/fields/view/tweet.hbt
+                ez-tweet-view:
+                    requires: ['ez-fieldview', 'tweetview-ez-template']
+                    dependencyOf: ['ez-rawcontentview']
+                    path: bundles/ezsystemstweetfieldtype/js/views/fields/ez-tweet-view.js
+```
+
+After these steps you should see correct information when viewing Content Item in "Content structure" tab in PlatformUI.
+
```

Comment:
Maybe worth repeating the warning about dumping assets (if it applies here as well)?